erpnext_documentation/www/docs/user/manual/en/manufacturing/downtime-entry.md

```diff
@@ -0,0 +1,10 @@
+<!-- add-breadcrumbs -->
+# Downtime Entry
+
+Downtime Entry can be found at:
+
+> Home > Manufacturing > Production > Downtime Entry
+
+<img class="screenshot" alt="Production Forecasting" src="{{docs_base_url}}/assets/img/manufacturing/downtime-entry.png">
+
+Downtime Entry document is used to capture the machine's Downtime in minutes. Using Downtime entry data operator knows which machine not performing well and needs a maintenance.
```

Comment:
Incomplete!
1. how does it capture? is it auto filled or user has to do it?
2. explain the fields if manual entry